API.Test/UserTest.cs

```diff
@@ -17,7 +17,7 @@ internal class UserTest
     [Test]
     public async Task GetCurrentPersonAsync()
     {
-        IUser user = await SetUp.Client.GetSignedInUserAsync();
+        IUser user = await SetUp.Client.GetSignedInUserAsync(); // Different values? May obsolete test?
```

Comment:
I don't really expect different values here but only to make sure that this method works.